.github/workflows/elixir.yml

```
@@ -15,8 +15,8 @@ jobs:
 
     strategy:
       matrix:
-        otp: [23.1, 22.3, 21.3]
-        elixir: [1.11, 1.10.4, 1.9.4, 1.8.2, 1.7.4]
+        otp: [24.0, 23.3.4, 22.3]
+        elixir: [1.11.4, 1.10.4, 1.9.4, 1.8.2]
 
```

We can make that version list a bit easier to maintain.
For OTPs we only track latest (minor,patch) for each major, and for elixir always the latest patch level per (major,minor).

```suggestion
        otp: [24. 23, 22]
        elixir: ['1.11', '1.10', '1.9', '1.8']
```

I see you truncated the elixir list from 5 to 4.
Any policy of how old an elixir version can be, respectively how many versions we want to test against?
Personally for less critical projects I opt for a maximum of 3 main versions; for languages with around a early cadence this is enough time and version support IMHO.